src/common/updating/updating.cpp

```diff
@@ -209,16 +183,19 @@ namespace updating
         return { std::move(path_str) };
     }
 
+    std::filesystem::path create_download_path(const std::wstring& filename)
+    {
+        auto installer_download_dst = get_pending_updates_path();
+        std::error_code _;
+        std::filesystem::create_directories(installer_download_dst, _);
+        installer_download_dst /= filename;
```

Comment:
this function is doing 2 things at once, making its name confusing. let's not pass `filename` and do `create_download_path() / new_version->installer_filename` instead